wg-charter.html

```diff
@@ -0,0 +1,387 @@
+            <p>
+                The GPU for the Web Community group will produce deliverables to be
+                used within this Working Group. In particular, a conformance test suite
+                and possibly reference implementations.
+            </p>
+        </div>
+      </section>
+
+
+      <section id="coordination">
+        <h2>Coordination</h2>
+        <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a> for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>. Invitation for review must be issued during each major standards-track document transition, including <a href="https://www.w3.org/2018/Process-20180201/#RecsWD" title="First Public Working Draft">FPWD</a> and <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Candidate Recommendation">CR</a>, and should be issued when major changes occur in a specification.</p>
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2018/Process-20180201/#WGCharter">W3C Process Document</a>:</p>
+
+        <div>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+            <dt><a href="https://www.w3.org/2001/tag/">Technical Architecture Group (TAG)</a></dt>
+        <dd>The WG may ask the Technical Architecture Group to review some of its specifications.</dd>
+
+        <dt><a href="https://www.w3.org/WebPlatform/WG/">Web Platform Working Group</a></dt>
+        <dd>This Working Group develops APIs for client-side development and for markup vocabularies for describing and controlling client-side application behavior.</dd>
+
+        <dt><a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a></dt>
+        <dd>This Working Group develops security and policy mechanisms to improve the security of Web Applications, and enable secure cross-site communication.</dd>
+
+        <dt><a href="https://www.w3.org/wasm/">Web Assembly Working Group</a></dt>
+        <dd>This Working Group develops a size- and load-time-efficient format and execution environment, allowing compilation to the web with consistent behavior across a variety of implementations.</dd>
+
+        <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator Community Group</a></dt>
+        <dd>
+          This group provides a lightweight venue for proposing, incubating and discussing new web platform features. The Web Performance Working group will incubate and review new proposals that are within scope of our charter within the WICG. Once such WICG-incubated proposal is implemented and available in at least one major browser, and has support from one more, it may be adopted by the Web Performance Working group.
+        </dd>
```

Comment:
@grorg, it may be useful to link back to https://webmachinelearning.github.io/charter/#coordination from here since participants of these two groups are already actively coordinating. Proposal:
```
<dt><a href="https://www.w3.org/community/webmachinelearning/">Machine Learning for the Web Community Group</a></dt>
<dd>This Community Group incubates a dedicated low-level Web API for machine learning inference.</dd>
```

Reply:
Good idea! Thanks @anssiko